keel-titus-plugin/keel-titus-plugin.gradle.kts

```diff
@@ -24,4 +24,5 @@ dependencies {
   testImplementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml")
   testImplementation("org.funktionale:funktionale-partials")
   testImplementation("org.apache.commons:commons-lang3")
+  testImplementation(project(":keel-core-test"))
```

Comment:
Not sure why this dependency is added when it looks like you did that to get access to `MutableClock` but then moved `MutableClock` to `keel-test`

Reply:
This module is needed now so that the titus plugin can implement the base cluster test